src/components/organisms/Table/Table.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FunctionComponent, useEffect } from 'react';
+import { createUseStyles } from 'react-jss';
+import type { Position } from '@models/Position';
+import { PlayingCard as PlayingCardComponent } from '@atoms/PlayingCard';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { initCardPositions, updateCardPosition } from '@store/slices/cards';
+import type { RootState } from '@store/index';
+import type { PlayingCardType } from '../../../models/PlayingCardType';
```

Comment:
You should be able to import it from `@models`

Reply:
Oops 🙃